plugins/flowdock/app/assets/javascripts/flowdock/application.js

```diff
@@ -1,112 +1,2 @@
 //= require jquery_mentions_input/jquery.elastic.source.js
```

Comment:
`elastic` does not seem to work on Chrome and Safari. Its last update was 5 years ago. http://unwrongest.com/projects/elastic/

We could remove it for now and figure out a different way to make the buddy approval message textarea looks nicer. I feel that setting a longer initial height would solve the most immediate UI issue.